CITATION.cff

```diff
@@ -1,20 +1,41 @@
 cff-version: 1.2.0
-message: "If you use this software, please cite it as below."
+message: |
+  Please cite this software using these metadata. "*" indicates co-equal authorship.
+
+title: "QGreenland"
+doi: "10.5281/zenodo.4558266"
+abstract: "This software generates the QGreenland data package for QGIS."
+contact:
+  - name: "The QGreenland Team"
+    email: "qgreenland.info@gmail.com"
+  - name: "The QGreenland community" 
+    website: "https://github.com/nsidc/qgreenland/discussions"
+type: "software"
+license: ["MIT"]
+keywords:
+  - "GIS"
+  - "QGIS"
+  - "Python"
+  - "education"
+
+url: "https://github.com/nsidc/qgreenland"
+repository-code: "https://github.com/nsidc/qgreenland"
+
+version: "3.0.0alpha4"
+date-released: "2023-08-23"
+
 authors:
-- family-names: "Moon"
-  given-names: "Twila"
-  orcid: "https://orcid.org/0000-0003-0968-7008"
-- family-names: "Fisher"
+- family-names: "*Fisher"
   given-names: "Matt"
   orcid: "https://orcid.org/0000-0003-3260-5445"
-- family-names: "Harden"
-  given-names: "Lynne"
-  orcid: "https://orcid.org/0000-0002-0505-6762"
-- family-names: "Stafford"
+  website: "https://github.com/mfisher87"
+- family-names: "*Stafford"
   given-names: "Trey"
   orcid: "https://orcid.org/0000-0002-5510-0055"
-title: "QGreenland"
-version: 3.0.0alpha4
-doi: 10.5281/zenodo.4558266
-date-released: 2021-02-23
-url: "https://github.com/nsidc/qgreenland"
+  website: "https://github.com/trey-stafford"
+- family-names: "Moon"
+  given-names: "Twila"
+  orcid: "https://orcid.org/0000-0003-0968-7008"
+- family-names: "Thurber"
+  given-names: "Alyse"
+  orcid: "https://orcid.org/0009-0003-2026-0002"
```

Comment:
@twilamoon-science @alysethurber I added Trey and my own GitHub profiles to our author entries with the `website` key. Do you guys have website you'd like to link here?